src/scene/export.ts

```diff
@@ -11,6 +11,7 @@ import {
   getInitializedImageElements,
   updateImageCache,
 } from "../element/image";
+import pkg from "../../src/packages/excalidraw/package.json";
```

Comment:
I'm not sure we want inlining `package.json` just to get `name` and `version`

Reply:
I was thinking of creating util out of it instead as this is useful in other places too

Reply:
either way, will most likely require a build step (similar to how we're handling `build-version.js`)

Reply:
I have introduced env vars to fix this, https://github.com/excalidraw/excalidraw/pull/5065/commits/d9159fa12e34dcf0361d4a03ccf4163fbc294f9e